graylog2-server/src/main/java/org/graylog/events/rest/EventDefinitionsResource.java

```diff
@@ -109,6 +112,7 @@ public EventDefinitionDto get(@ApiParam(name = "definitionId") @PathParam("defin
     @ApiOperation("Create new event definition")
     @AuditEvent(type = EventsAuditEventTypes.EVENT_DEFINITION_CREATE)
```

Comment:
When we don't need to check for a specific instance, like here for creation, we can use the annotation instead of manually calling `checkPermissions()`.

```suggestion
     @AuditEvent(type = EventsAuditEventTypes.EVENT_DEFINITION_CREATE)
     @RequiresPermissions(RestPermissions.EVENT_DEFINITIONS_CREATE)
```